### PR TITLE
Reflect dependencies moved to Voxpupuli

### PR DESCRIPTION
**Pull request by jay7x**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
       "version_requirement": ">= 2.4.0 < 4.0.0"
     },
     {
-      "name": "herculesteam-augeasproviders_sysctl",
+      "name": "puppet-augeasproviders_sysctl",
```
Comment (bastelfreak) [inline metadata.json:28]: please also update https://github.com/voxpupuli/puppet-k8s/blob/master/.fixtures.yml to pull the correct repo (from git, not the forge)

Comment (jay7x) [reply metadata.json:28]: good catch!
I see camptocamp/systemd there.. shall I update it too?

Comment (jay7x) [reply metadata.json:28]: Updated

Comment (bastelfreak) [reply metadata.json:28]: yes please

**Comment by jay7x** [comment]
⚠️ I hit the following when trying to use the module with the changes:
```
Unable to resolve modules: Puppetfile Resolver could not find compatible versions for possibility named "augeasproviders_sysctl":
  In Puppetfile:
    -k8s >= 0 was resolved to Git puppet-k8s-0.0.1, which depends on
      puppet-augeasproviders_sysctl >= 2.6.2 < 3.0.0
```

**Pull request closed without merging**